no-computer.md

```diff
@@ -11,41 +11,106 @@ _What need is met by your idea? Who are you helping? What is that person's
+The board games involve the players sending in instructions for their units
+on the board.
+These instructions are collected then played out simultaneously.
+This causes a lot of conflicts between their instructions which are then 
+resolved based on the game's rules.
+Further complications arise when players send invalid instructions.
+
+A program that can take in these instructions and then validate moves and
+resolve conflicts would speed up this process.
+As of now, it takes 5 to 10 minutes for people to manage this process.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This program would need to take in inputs from the players.
+A DSL would be an appropriate method to take in these instructions.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I like boardgames and I'm somewhat interested in boardgame design. 
+This project builds on a homebrewed version of Diplomacy, so I'm excited to
+contribute to a project that already exist and needs help.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Multiplayer turn-based homebrewed boardgames.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Diplomacy has its own syntax for user instructions
+```
+<Nation>
+<Boardgame piece> <Location> to <Destination>
+
+<Nation>
+<Boardgame piece> <Location> holds
+
+<Nation>
+<Boardgame piece> <Location> supports <Boardgame piece> <Location>
+```
+Since this is the syntax that players use, a DSL for the game would also use
+the same syntax to cater to its users.
+
+But the DSL would need to know what the map looks like to validate moves.
+We could use an adjacency matrix to store this data.
+The players could enter the data for this matrix using a spreadsheet.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would store the representation of the graph.
+This map will be used later for validating players' instructions and resolving
+conflicts.
+
+As players send in orders, it would read the moves to ensure that they are
+valid.
+When the order collecting phase is over, it would then collect the instructions 
+from each player.
+It would go through the instructions according to the game's rules and resolves
+conflicts.
+Finally, it would output the final positions and states of each boardgame piece.
+The output format could be a list of every piece and its respective location.
+
+If the program finds an error in a player's set of instructions it could send
+an email to the player based on a database of players' information.
+An email might be the best way of notifying the player since each player's
+instructions should be kept secret from other players.
+
```

Comment:
The idea of secrecy intrigues me. I like email (or text..) as a private messaging system. Maybe an alert to the DSL on that users computer could also suffice.